Trophy case should accept individual commit links

Hi! I came across zizmor when I heard about Ultralytics, and I used zizmor on my [repo](https://github.com/girlbossceo/conduwuit) with 573 stars, and I would like to add it to the trophy case but it only supports PR links where as I typically do a lot of work in one PR and merge it (we're a fast-moving project).

Is it possible for the trophy case to support adding a commit link like this?

https://github.com/girlbossceo/conduwuit/pull/636/commits/c6bf8f5ea15a6b963220a1de4bb50a639d0d0696

It looks like it just gets skipped over by `render-trophies.py` no matter which way I put it.

Adding this in trophies.txt: `girlbossceo/conduwuit/commit/c6bf8f5ea15a6b963220a1de4bb50a639d0d0696`
Outputs:
```
❯ make trophies
make: Nothing to be done for `trophies'.
```

Adding the full URL outputs:
```
uv run --no-project docs/snippets/render-trophies.py > docs/snippets/trophies.md
Traceback (most recent call last):
  File "/Users/strawberry/zizmor/docs/snippets/render-trophies.py", line 23, in <module>
    org, rest = trophy.split("/")
    ^^^^^^^^^
ValueError: too many values to unpack (expected 2)
make: *** [docs/snippets/trophies.md] Error 1
```

--- a/docs/snippets/render-trophies.py
+++ b/docs/snippets/render-trophies.py
@@ -5,7 +5,6 @@
 
 from pathlib import Path
 
-
 _TROPHIES = Path(__file__).parent / "trophies.txt"
 
 _TEMPLATE = """
@@ -21,7 +20,11 @@ for trophy in sorted(_TROPHIES.open().readlines()):
         continue
 
     org, rest = trophy.split("/")
-    repo, _ = rest.split("#")
+    if "#" in rest:
+        repo, _ = rest.split("#")
+    else:
+        repo, _ = rest.split("@")
+
     # NOTE: We request 40x40 from GitHub, but sometimes it gives us a bigger one.
     # Consequently, we also style with `width` to keep things consistent.
     print(_TEMPLATE.format(org=org, repo=repo, trophy=trophy))

--- a/docs/snippets/trophies.md
+++ b/docs/snippets/trophies.md
@@ -71,6 +71,12 @@
 
     ethereum/hevm#615
 
+-   ![](https://github.com/girlbossceo.png?size=40){ width="40" loading=lazy align=left } girlbossceo/conduwuit
+
+    ---
+
+    girlbossceo/conduwuit@c6bf8f5ea15a6b963220a1de4bb50a639d0d0696
+
 -   ![](https://github.com/hugovk.png?size=40){ width="40" loading=lazy align=left } hugovk/em-keyboard
 
     ---

--- a/docs/snippets/trophies.txt
+++ b/docs/snippets/trophies.txt
@@ -1,12 +1,14 @@
 # one per line, order is not important
-# trophies MUST be formatted as owner/repo#number,
-# where owner/repo is the GitHub repo slug and
-# number is the issue/PR that introduces or uses zizmor
+
+# two formats are supported:
+# owner/repo#number for PRs
+# owner/repo@commit for individual commits
 
 adafruit/circuitpython#9785
 astral-sh/ruff#14844
 astropy/astropy#17315
 cakephp/cakephp#18081
+girlbossceo/conduwuit@c6bf8f5ea15a6b963220a1de4bb50a639d0d0696
 danmar/cppcheck#7044
 DataDog/datadog-agent#30871
 Diaoul/subliminal#1190

